views/js/controller/creator/views/itemref.js

```diff
@@ -20,24 +20,45 @@
  * @author Bertrand Chevrier <bertrand@taotesting.com>
  */
 define([
-'jquery',
-'lodash', 
-'i18n',
-'taoQtiTest/controller/creator/views/actions',
-'taoQtiTest/controller/creator/helpers/sectionCategory'],
-function($, _, __, actions, sectionCategory){
+    'jquery',
+    'lodash',
+    'i18n',
+    'taoQtiTest/controller/creator/views/actions',
+    'taoQtiTest/controller/creator/helpers/sectionCategory',
+    'taoQtiTest/controller/creator/helpers/qtiTest',
+    'tpl!taoQtiTest/controller/creator/templates/itemref-props-weight'
+],
+function(
+    $,
+    _,
+    __,
+    actions,
+    sectionCategory,
+    qtiTestHelper,
+    weightTpl
+){
     'use strict';
 
-   /**
-    * Set up an item ref: init action beahviors. Called for each one.
-    *
-    * @param {jQueryElement} $itemRef - the itemRef element to set up
-    * @param {Object} model - the data model to bind to the ref
-    */
-   var setUp =  function setUp ($itemRef, model){
+    /**
+     * We need to resize the itemref in case of long labels
+     */
+    var resize = _.throttle(function resize(){
```

Comment:
The function was there and has been left untouched. I just moved it to remove an eslint warning. Thus, I would prefer to address this issue in a separate PR.